articles/stream-analytics/stream-analytics-parsing-json.md

```diff
@@ -110,42 +132,158 @@ FROM input as event
 CROSS APPLY GetRecordProperties(event.SensorReadings) AS sensorReading
 ```
 
+The result is:
+
+|DeviceID|SensorName|AlertMessage|
+|-|-|-|
+|12345|Temperature|80|
+|12345|Humidity|70|
+|12345|CustomSensor01|5|
+|12345|CustomSensor02|99|
+|12345|SensorMetadata|[object Object]|
 
+Using [WITH](https://docs.microsoft.com/en-us/stream-analytics-query/with-azure-stream-analytics), it's then possible to route those events to different destinations:
```

Comment:
Please remove `en-us` from all links. The system will pick the locale according to the user's preferences.